services/web/server/src/simcore_service_webserver/log.py

```diff
@@ -18,13 +18,18 @@
 )
 
 
-def setup_logging(*, level: str | int, slow_duration: float | None = None):
+def setup_logging(
+    *,
+    level: str | int,
+    slow_duration: float | None = None,
+    log_format_local_dev_enabled: bool = False
```

Comment:
always wondering at these defaults. in this case do we really need them? there is just the one call from cli.py right? why not just remove all the defaults if they are anyway set?